src/temp/ProjectImagesSlider/index.tsx

```diff
@@ -19,8 +19,8 @@ const ProjectImagesSlider = ({ images, height, imageSize, type }: Props) => {
   useEffect(() => {
     images.forEach((sliderImage) => {
       if (sliderImage.image) {
-        const imageURL = getImageUrl(type, imageSize, sliderImage.image);
-
+        // const imageURL = getImageUrl(type, imageSize, sliderImage.image);
+        const imageURL = sliderImage.image; //to be replaced with above line
```

Comment:
If we do this, component will be rendered correctly for either the story or the actual app. We can achieve both by checking for an absolute URL and deciding whether to call `getImageUrl` accordingly.